src/autoslope.h

```diff
@@ -34,6 +34,27 @@ inline bool AutoslopeCheckForEntranceEdge(TileIndex tile, int z_new, Slope tileh
 	return ((tileh_new == SLOPE_FLAT) || CanBuildDepotByTileh(entrance, tileh_new));
 }
 
+/**
+ * Autoslope check for tiles with something build along an axis.
```

Comment:
Sorry for the tiniest nitpick. All else looks good!

```suggestion
 * Autoslope check for tiles with something built along an axis.
```